README.md

```diff
@@ -3,192 +3,142 @@
+#### Get a cookie
 
-hasCookie('name', options); // => true
-hasCookie('nothing', options); // => false
+```javascript
+const value = getCookie('key', options);
 ```
 
-Delete a cookie:
+#### Get all cookies
 
-```js
-import { deleteCookie } from 'cookies-next';
-
-deleteCookie(name, options);
+```javascript
+const cookies = getCookies(options);
 ```
 
-_IMPORTANT! When deleting a cookie and you're not relying on the default attributes,
-you must pass the exact same path and domain attributes that were used to set the cookie:_
-
-```js
-import { deleteCookie } from 'cookies-next';
+#### Check if a cookie exists
 
-deleteCookie(name, { path: '/path', domain: '.yourdomain.com' });
+```javascript
+const exists = hasCookie('key', options);
 ```
 
-### Performance
-
-The [time complexity](https://en.wikipedia.org/wiki/Time_complexity) of all operations is linear with the number of cookies.
-For example, under the hood, `getCookie` calls `getCookies`. When working reading multiple cookies,
-it is fastest to use `getCookies` and inspect the returned object.
+#### Delete a cookie
 
-## Client and Server
-
-If you pass ctx (Next.js context) in function, then this function will be done on both client and server
-
-If the function should be done only on client or can't get ctx, pass null or {}
-as the first argument to the function and when server side rendering, this function return undefined;
-
-In Next.js 13+, you can [read(only)](https://nextjs.org/docs/app/api-reference/functions/cookies) cookies in `Server Components` and read/update them in `Server Actions`. This can be achieved by using the `cookies` function as an option, which is imported from `next/headers`, instead of using `req` and `res`.
-
-#### Client - App Router Example
-
-```ts
-'use client';
-import { getCookies, setCookie, deleteCookie, getCookie } from 'cookies-next';
-
-getCookies();
-getCookie('key');
-setCookie('key', 'value');
-deleteCookie('key');
+```javascript
+deleteCookie('key', options);
 ```
 
-#### Client - Pages Router Example
+### Client-side Usage
 
-```js
+```javascript
+'use client';
 import { getCookies, setCookie, deleteCookie, getCookie } from 'cookies-next';
 
-// we can use it anywhere
+// Use anywhere in client-side code
 getCookies();
 getCookie('key');
 setCookie('key', 'value');
 deleteCookie('key');
 ```
 
-#### SSR - App Router Example
-
-`/app/page.tsx`
-
-```tsx
-import { setCookie, getCookie, getCookies, deleteCookie, hasCookie } from 'cookies-next';
-import { cookies } from 'next/headers';
-
-const Home = async () => {
-  // It's not possible to update the cookie in RSC
-  ❌ setCookie("test", "value", { cookies }); 👉🏻// Won't work.
-  ❌ deleteCookie('test1', { cookies }); 👉🏻// Won't work.
-
-  ✔️ getCookie('test1', { cookies });
-  ✔️ getCookies({ cookies });
-  ✔️ hasCookie('test1', { cookies });
-
-  return (
-    <main>
-      <h1>Hello cookies next</h1>
-    </main>
-  );
-};
-
-export default Home;
-
-```
-
```

Comment:
I would keep the section explaining what works and what doesn't. People often mistake this behavior for a bug, and this section does a great job of clarifying it.